data/data-pipeline/data_pipeline/tests/score/test_output.py

```diff
@@ -203,3 +212,44 @@ def test_donut_hole_addition_to_score_n(final_score_df):
     assert (
         new_donuts > 0
     ), "FYI: The adjacency index is doing nothing. Consider removing it?"
+
+
+def test_data_sources(
+    final_score_df, hud_housing_df, ejscreen_df, cdc_places_df
+):
+    data_sources = {
+        key: value for key, value in locals().items() if key != "final_score_df"
+    }
+
+    for data_source_name, data_source in data_sources.items():
+        final = "_final"
+        df: pd.DataFrame = final_score_df.merge(
+            data_source,
+            on=GEOID_TRACT_FIELD_NAME,
+            indicator="MERGE",
+            suffixes=(final, f"_{data_source_name}"),
+            how="left",
+        )
+        data_source_columns = [
```

Comment:
I think here it might actually be easier to do the set intersection of the columns, but that's a REAL nit

Reply:
I can def at least pull the duplicated lines out, now that I look at it again.